regression/goto-cc-file-local/chain.sh

```diff
@@ -54,39 +54,61 @@ else
   export_flag="--export-file-local-symbols"
 fi
 
-cnt=0
-for src in ${SRC}; do
-  cnt=$((cnt + 1))
-  out_suffix=""
-  if is_in out-file-counter "$ALL_ARGS"; then
-    out_suffix="_$cnt"
-    if is_in suffix "$ALL_ARGS"; then
-      suffix="--mangle-suffix custom_$cnt"
+OUT_FILE="main.gb"
+if is_in compile-and-link "$ALL_ARGS"; then
+    if [[ "${is_windows}" == "true" ]]; then
+      "${goto_cc}"                        \
+          ${export_flag}                  \
+          --verbosity 10                  \
+          ${wall}                         \
+          ${suffix}                       \
+          ${SRC}                          \
+          /Fo"${OUT_FILE}"
```

Comment:
I believe this should be `/Fe` rather than `/Fo`, since we're compiling and linking at the same time